vsintegration/src/fsharp.vsreferences.targets

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Most of the vsintegration stuff already imports FSharpSource.targets. It would be ideal if we could keep targets/settings like this together.